PR #24 non stable

Le PR #24 se compile bien, mais je déploiement sur le serveur ne passe pas.

Ci-dessous les traces du serveur : 
[server.log](https://github.com/GIP-GRADeS-BFC/SORMAS-Project/files/4869408/server.log)

On remarque cela dans les logs : 
Caused by: javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not execute statement
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:154)
	at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:181)
	at org.hibernate.query.internal.AbstractProducedQuery.executeUpdate(AbstractProducedQuery.java:1617)
	at de.symeda.sormas.backend.common.StartupShutdownService.updateDatabase(StartupShutdownService.java:492)
	at de.symeda.sormas.backend.common.StartupShutdownService.startup(StartupShutdownService.java:155)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.sun.ejb.containers.interceptors.BeanCallbackInterceptor.intercept(InterceptorManager.java:1047)
	at com.sun.ejb.containers.interceptors.CallbackChainImpl.invokeNext(CallbackChainImpl.java:72)
	at com.sun.ejb.containers.interceptors.CallbackInvocationContext.proceed(CallbackInvocationContext.java:206)
	at com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)
	at com.sun.ejb.containers.interceptors.SystemInterceptorProxy.init(SystemInterceptorProxy.java:125)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.sun.ejb.containers.interceptors.CallbackInterceptor.intercept(InterceptorManager.java:998)
	at com.sun.ejb.containers.interceptors.CallbackChainImpl.invokeNext(CallbackChainImpl.java:72)
	at com.sun.ejb.containers.interceptors.CallbackInvocationContext.proceed(CallbackInvocationContext.java:206)
	at org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:81)
	at org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)
	at sun.reflect.GeneratedMethodAccessor133.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.sun.ejb.containers.interceptors.CallbackInterceptor.intercept(InterceptorManager.java:998)
	at com.sun.ejb.containers.interceptors.CallbackChainImpl.invokeNext(CallbackChainImpl.java:72)
	at com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:417)
	at com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:380)
	at com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:2087)
	at com.sun.ejb.containers.AbstractSingletonContainer.createSingletonEJB(AbstractSingletonContainer.java:555)
	... 38 more
Caused by: org.hibernate.exception.SQLGrammarException: could not execute statement
	at org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:106)
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:99)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:178)
	at org.hibernate.engine.query.spi.NativeSQLQueryPlan.performExecuteUpdate(NativeSQLQueryPlan.java:107)
	at org.hibernate.internal.SessionImpl.executeNativeUpdate(SessionImpl.java:1632)
	at org.hibernate.query.internal.NativeQueryImpl.doExecuteUpdate(NativeQueryImpl.java:295)
	at org.hibernate.query.internal.AbstractProducedQuery.executeUpdate(AbstractProducedQuery.java:1608)
	... 67 more
Caused by: org.postgresql.util.PSQLException: ERROR: relation "action" already exists
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2468)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2211)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:309)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:446)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:370)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:149)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.postgresql.ds.PGPooledConnection$StatementHandler.invoke(PGPooledConnection.java:428)
	at com.sun.proxy.$Proxy557.executeUpdate(Unknown Source)
	at com.sun.gjc.spi.base.PreparedStatementWrapper.executeUpdate(PreparedStatementWrapper.java:127)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.sun.gjc.spi.jdbc40.ProfiledConnectionWrapper40$1.invoke(ProfiledConnectionWrapper40.java:437)
	at com.sun.proxy.$Proxy558.executeUpdate(Unknown Source)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:175)
	... 71 more

Quels éléments de la base de données avez-vous modifié entre ce déploiement et le précédent?

--- a/sormas-api/src/main/java/de/symeda/sormas/api/event/EventDto.java
+++ b/sormas-api/src/main/java/de/symeda/sormas/api/event/EventDto.java
@@ -50,7 +50,7 @@ public class EventDto extends EntityDto {
 	public static final String DISEASE_DETAILS = "diseaseDetails";
 	public static final String SURVEILLANCE_OFFICER = "surveillanceOfficer";
 	public static final String TYPE_OF_PLACE_TEXT = "typeOfPlaceText";
-	public static final String NOM_TYPE_OF_PLACE = "nomTypeOfPlace";
+	public static final String NAME_TYPE_OF_PLACE = "nameTypeOfPlace";
 	public static final String REPORT_LAT = "reportLat";
 	public static final String REPORT_LON = "reportLon";
 	public static final String TYPE_OF_RISK = "typeOfRisk";
@@ -78,7 +78,7 @@ public class EventDto extends EntityDto {
 	private Double reportLat;
 	private Double reportLon;
 	private Float reportLatLonAccuracy;
-	private String nomTypeOfPlace;
+	private String nameTypeOfPlace;
 	private TypeOfRisk typeOfRisk;
 
 	public static EventDto build() {
@@ -256,11 +256,11 @@ public class EventDto extends EntityDto {
 		return new EventReferenceDto(getUuid());
 	}
 
-	public String getNomTypeOfPlace() {
-		return nomTypeOfPlace;
+	public String getNameTypeOfPlace() {
+		return nameTypeOfPlace;
 	}
 
-	public void setNomTypeOfPlace(String nomTypeOfPlace) {
-		this.nomTypeOfPlace = nomTypeOfPlace;
+	public void setNameTypeOfPlace(String nameTypeOfPlace) {
+		this.nameTypeOfPlace = nameTypeOfPlace;
 	}
 }

--- a/sormas-api/src/main/java/de/symeda/sormas/api/event/TypeOfPlace.java
+++ b/sormas-api/src/main/java/de/symeda/sormas/api/event/TypeOfPlace.java
@@ -45,7 +45,7 @@ public enum TypeOfPlace {
 	PROFESSIONAL_CIRCLES(new Locale("fr", "FR")),
 	EVENT(new Locale("fr", "FR")),
 	SMALL_GEOGRAPHIC_UNIT(new Locale("fr", "FR")),
-	SPECIFIC_COMMUNAUTY(new Locale("fr", "FR")),
+	SPECIFIC_COMMUNITIES(new Locale("fr", "FR")),
 	OTHER(new Locale("en"));
 
 	private Locale locale;

--- a/sormas-api/src/main/resources/captions_fr-FR.properties
+++ b/sormas-api/src/main/resources/captions_fr-FR.properties
@@ -764,9 +764,6 @@ EpiDataTravel.travelType=Type de voyage
 
 # Event
 eventEventActions=Actions de l'événement
-eventEventParticipants=Participants à l'événement
-eventEventsList=Liste d'événements
-eventNewEvent=Nouvel événement
 eventActiveEvents=Événements actifs
 eventArchivedEvents=Événements archivés
 eventAllEvents=Tous les événements
@@ -799,7 +796,7 @@ Event.srcOrigin=Origine
 Event.surveillanceOfficer=Agent de surveillance responsable
 Event.typeOfPlace=Type de lieu
 Event.typeOfPlaceText=Spécifier le type de lieu
-Event.nomTypeOfPlace=Nom de la collectivité 
+Event.nameTypeOfPlace=Nom de la collectivité 
 Event.uuid=ID d’événement
 
 # EventParticipant

--- a/sormas-backend/src/main/java/de/symeda/sormas/backend/event/Event.java
+++ b/sormas-backend/src/main/java/de/symeda/sormas/backend/event/Event.java
@@ -75,7 +75,7 @@ public class Event extends CoreAdo {
 	public static final String REPORT_LAT = "reportLat";
 	public static final String REPORT_LON = "reportLon";
 	public static final String ARCHIVED = "archived";
-	public static final String NOM_TYPE_OF_PLACE = "nomTypeOfPlace";
+	public static final String NAME_TYPE_OF_PLACE = "nameTypeOfPlace";
 	public static final String TYPE_OF_RISK = "typeOfRisk";
 
 	private EventStatus eventStatus;
@@ -98,7 +98,7 @@ public class Event extends CoreAdo {
 	private Double reportLat;
 	private Double reportLon;
 	private Float reportLatLonAccuracy;
-	private String nomTypeOfPlace;
+	private String nameTypeOfPlace;
 	private TypeOfRisk typeOfRisk;
 
 	private boolean archived;
@@ -321,11 +321,11 @@ public class Event extends CoreAdo {
 	}
 
 	@Column(length = COLUMN_LENGTH_DEFAULT)
-	public String getNomTypeOfPlace() {
-		return nomTypeOfPlace;
+	public String getNameTypeOfPlace() {
+		return nameTypeOfPlace;
 	}
 
-	public void setNomTypeOfPlace(String nomTypeOfPlace) {
-		this.nomTypeOfPlace = nomTypeOfPlace;
+	public void setNameTypeOfPlace(String nameTypeOfPlace) {
+		this.nameTypeOfPlace = nameTypeOfPlace;
 	}
 }

--- a/sormas-backend/src/main/java/de/symeda/sormas/backend/geocoding/GeocodingServiceSirenFrench.java
+++ b/sormas-backend/src/main/java/de/symeda/sormas/backend/geocoding/GeocodingServiceSirenFrench.java
@@ -78,7 +78,7 @@ public class GeocodingServiceSirenFrench {
 
         WebTarget target = client.target(url);
         Response response = target.request(MediaType.APPLICATION_JSON_TYPE)
-                .header(HttpHeaders.AUTHORIZATION, "Bearer " + "c5173994-3163-3bcf-a39c-7afc0d8dd187")
+                .header(HttpHeaders.AUTHORIZATION, "Bearer " + "c305032c-eaa7-3d47-bdcc-ed9c62e68aa1")
                 .get();
 
         if (response.getStatusInfo().getFamily() != Response.Status.Family.SUCCESSFUL) {

--- a/sormas-backend/src/main/resources/sql/sormas_schema.sql
+++ b/sormas-backend/src/main/resources/sql/sormas_schema.sql
@@ -4717,5 +4717,5 @@ INSERT INTO schema_version (version_number, comment) VALUES (220, 'Add field typ
              
 ALTER TABLE events ADD nameTypeOfPlace varchar(512);
 ALTER TABLE events_history ADD nameTypeOfPlace varchar(512);
-INSERT INTO schema_version (version_number, comment) VALUES (221, 'Add field nom type of place of an event');
+INSERT INTO schema_version (version_number, comment) VALUES (221, 'Add field name type of place of an event');
 -- *** Insert new sql commands BEFORE this line ***

--- a/sormas-backend/src/main/resources/sql/sormas_schema.sql
+++ b/sormas-backend/src/main/resources/sql/sormas_schema.sql
@@ -4717,4 +4717,5 @@ INSERT INTO schema_version (version_number, comment) VALUES (220, 'Add field typ
              
 ALTER TABLE events ADD nomTypeOfPlace varchar(512);
 ALTER TABLE events_history ADD nomTypeOfPlace varchar(512);
+INSERT INTO schema_version (version_number, comment) VALUES (221, 'Add field nom type of place of an event');
 -- *** Insert new sql commands BEFORE this line ***

--- a/sormas-backend/src/main/resources/sql/sormas_schema.sql
+++ b/sormas-backend/src/main/resources/sql/sormas_schema.sql
@@ -4715,7 +4715,7 @@ ALTER TABLE events ADD COLUMN typeofrisk character varying(255);
 ALTER TABLE events_history ADD COLUMN typeofrisk character varying(255);
 INSERT INTO schema_version (version_number, comment) VALUES (220, 'Add field type of risk of an event');
              
-ALTER TABLE events ADD nomTypeOfPlace varchar(512);
-ALTER TABLE events_history ADD nomTypeOfPlace varchar(512);
+ALTER TABLE events ADD nameTypeOfPlace varchar(512);
+ALTER TABLE events_history ADD nameTypeOfPlace varchar(512);
 INSERT INTO schema_version (version_number, comment) VALUES (221, 'Add field nom type of place of an event');
 -- *** Insert new sql commands BEFORE this line ***

--- a/sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventDataForm.java
+++ b/sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventDataForm.java
@@ -89,7 +89,7 @@ public class EventDataForm extends AbstractEditForm<EventDto> {
 			fluidRowLocs(EventDto.SRC_TEL_NO, EventDto.SRC_EMAIL) +
 
 			loc(LOCATION_HEADING_LOC) +
-			fluidRowLocs(EventDto.TYPE_OF_PLACE, EventDto.TYPE_OF_PLACE_TEXT, EventDto.NOM_TYPE_OF_PLACE) +
+			fluidRowLocs(EventDto.TYPE_OF_PLACE, EventDto.TYPE_OF_PLACE_TEXT, EventDto.NAME_TYPE_OF_PLACE) +
 			fluidRowLocs(EventDto.EVENT_LOCATION) +
 			fluidRowLocs("", EventDto.SURVEILLANCE_OFFICER);
 	//@formatter:on
@@ -150,7 +150,7 @@ public class EventDataForm extends AbstractEditForm<EventDto> {
 		ComboBox typeOfPlace = addField(EventDto.TYPE_OF_PLACE, ComboBox.class);
 		typeOfPlace.setNullSelectionAllowed(true);
 		addField(EventDto.TYPE_OF_PLACE_TEXT,TextField.class);
-		addField(EventDto.NOM_TYPE_OF_PLACE, ComboBox.class);
+		addField(EventDto.NAME_TYPE_OF_PLACE, ComboBox.class);
 		addField(EventDto.REPORT_DATE_TIME, DateTimeField.class);
 		addField(EventDto.REPORTING_USER, ComboBox.class);
 		TextField srcOrigin = addField(EventDto.SRC_ORIGIN, TextField.class);
@@ -162,7 +162,7 @@ public class EventDataForm extends AbstractEditForm<EventDto> {
 		setReadOnly(true, EventDto.UUID, EventDto.REPORT_DATE_TIME, EventDto.REPORTING_USER);
 
 		FieldHelper.setVisibleWhen(getFieldGroup(), EventDto.TYPE_OF_PLACE_TEXT, EventDto.TYPE_OF_PLACE, Arrays.asList(TypeOfPlace.OTHER), true);
-		FieldHelper.setVisibleWhen(getFieldGroup(), EventDto.NOM_TYPE_OF_PLACE, EventDto.TYPE_OF_PLACE, Arrays.asList(TypeOfPlace.PROFESSIONAL_CIRCLES,TypeOfPlace.EHPAD,TypeOfPlace.SCHOOLS_UNIVERSITIES, TypeOfPlace.PENITENTIARY_ESTABLISHMENTS), true);
+		FieldHelper.setVisibleWhen(getFieldGroup(), EventDto.NAME_TYPE_OF_PLACE, EventDto.TYPE_OF_PLACE, Arrays.asList(TypeOfPlace.PROFESSIONAL_CIRCLES,TypeOfPlace.EHPAD,TypeOfPlace.SCHOOLS_UNIVERSITIES, TypeOfPlace.PENITENTIARY_ESTABLISHMENTS), true);
 
 		FieldHelper.setVisibleWhen(getFieldGroup(), Arrays.asList(EventDto.DISEASE_DETAILS), EventDto.DISEASE, Arrays.asList(Disease.OTHER), true);
 		FieldHelper.setRequiredWhen(getFieldGroup(), EventDto.DISEASE, Arrays.asList(EventDto.DISEASE_DETAILS), Arrays.asList(Disease.OTHER));
@@ -170,8 +170,8 @@ public class EventDataForm extends AbstractEditForm<EventDto> {
 		setRequired(true, EventDto.EVENT_STATUS, EventDto.UUID, EventDto.EVENT_DESC, EventDto.REPORT_DATE_TIME, EventDto.REPORTING_USER);
 		setTypeOfPlaceTextRequirement();
 
-		ComboBox nomTypeOfPlaceField = (ComboBox) getFieldGroup().getField(EventDto.NOM_TYPE_OF_PLACE);
-		nomTypeOfPlaceField.setTextInputAllowed(true);
+		ComboBox nameTypeOfPlaceField = (ComboBox) getFieldGroup().getField(EventDto.NAME_TYPE_OF_PLACE);
+		nameTypeOfPlaceField.setTextInputAllowed(true);
 		typeOfPlace.addValueChangeListener(e -> {
 			String value = "";
 			List<String> resultList = new ArrayList<>();
@@ -181,7 +181,7 @@ public class EventDataForm extends AbstractEditForm<EventDto> {
 			if(typeOfPlace.getValue() == TypeOfPlace.PROFESSIONAL_CIRCLES) {
 				resultList = FacadeProvider.getGeocodingFacadeFrench().getSireneEntrepriseAutoComplete(value);
 			}
-			FieldHelper.updateItems(nomTypeOfPlaceField, resultList);
+			FieldHelper.updateItems(nameTypeOfPlaceField, resultList);
 		});
 
 
@@ -215,24 +215,4 @@ public class EventDataForm extends AbstractEditForm<EventDto> {
 		typeOfPlaceField.addValueChangeListener(event -> typeOfPlaceTextField.setRequired(typeOfPlaceField.getValue() == TypeOfPlace.OTHER));
 	}
 
-	@SuppressWarnings("rawtypes")
-	public void setNomTypeOfPlaceRequirementUniversity(AutocompleteTextField nomTypeOfPlace){
-		FieldGroup fieldGroup = getFieldGroup();
-		ComboBox typeOfPlaceField = (ComboBox) fieldGroup.getField(EventDto.TYPE_OF_PLACE);
-		((AbstractField) typeOfPlaceField).setImmediate(true);
-
-		nomTypeOfPlace.setRequired(typeOfPlaceField.getValue() == TypeOfPlace.SCHOOLS_UNIVERSITIES);
-		typeOfPlaceField.addValueChangeListener(event -> nomTypeOfPlace.setRequired(typeOfPlaceField.getValue() == TypeOfPlace.SCHOOLS_UNIVERSITIES));
-
-	}
-
-	@SuppressWarnings("rawtypes")
-	public void setNomTypeOfPlaceRequirementCompany(AutocompleteTextField nomTypeOfPlace){
-		FieldGroup fieldGroup = getFieldGroup();
-		ComboBox typeOfPlaceField = (ComboBox) fieldGroup.getField(EventDto.TYPE_OF_PLACE);
-		((AbstractField) typeOfPlaceField).setImmediate(true);
-
-		nomTypeOfPlace.setRequired(typeOfPlaceField.getValue() == TypeOfPlace.PROFESSIONAL_CIRCLES);
-		typeOfPlaceField.addValueChangeListener(event -> nomTypeOfPlace.setRequired(typeOfPlaceField.getValue() == TypeOfPlace.PROFESSIONAL_CIRCLES));
-	}
 }

